index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Personal portfolio</title>
+
+  <link rel="Stylesheet" href="./styles.css" />
+</head>
+
+
+<body>
+  <header class="main-header">
+    <a href="#" class="logo">My Logo</a>
+    <nav class="nav">
+      <img src="/media/Icon-Menu.png" alt="Union" width="30px">
+     <!-- <ul class="nav-menu">
+       
+        
+        <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+        <li class="nav-link active"><a href="#About">About</a></li>
+        <li class="nav-link"><a href="#form">Contact</a></li>
+        
+
+        
+        
+      </ul>
+      !-->
+    </nav>
+  </header>
+
+  <main class="container">
+    <section class="main-section">
+      <div class="main-section-title">
+        <h1>I'm Gilbert, Glad to See You!</h1>
+       <!-- <h2>
+        I'm a software developer
+       </h2>
+      -->
+        
+      </div>
+      <p>
+        I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] Please ensure that this paragraph is fully visible on your app, it has no overflow, and it does not scroll on the page. You can achieve this by setting the height to `auto` and making the overflow visible.
  
![review1](https://user-images.githubusercontent.com/87394385/192094323-2398710d-e7b8-45c2-9920-4dcdc1fb0380.JPG)